src/figuresClasses.ts

```diff
@@ -1,11 +1,101 @@
-export interface Figure {}
+export interface Figure {
+  shape: `triangle` | `circle` | `rectangle`;
+  color: `red` | `green` | `blue`;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    a: number,
+    b: number,
+    c: number,
+  ) {
+    if (a === 0 || b === 0 || c === 0) {
+      throw new Error('Sides can not be equal to zero');
+    }
+
+    if (a + b <= c || c + b <= a || a + c <= b) {
+      throw new Error(
+        'the longest side of a triangle is >= than a sum of two others',
+      );
```

Comment:
[CODE STYLE] - The error message could be more descriptive and follow the standard mathematical terminology. Instead of 'the longest side of a triangle is >= than a sum of two others', consider using 'The sum of any two sides of a triangle must be greater than the third side.'

Reply:
а